hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSet.java

```diff
@@ -208,6 +210,7 @@ public int partitions() {
     public DataSet<V> distinct(String name, boolean isLast, OperationContext context, boolean pushScope, String scopeDetail) {
         pushScopeIfNeeded(context, pushScope, scopeDetail);
         try {
+            int numPartitions = SparkUtils.getPartitions(rdd, defaultPartitions);
```

Comment:
The variable numPartitions is not used, why introduced here?

Reply:
Thanks, removed.